common/src/main/cpp/extensions/openxr_fb_passthrough_extension_wrapper.cpp

```diff
@@ -65,12 +67,47 @@ OpenXRFbPassthroughExtensionWrapper::~OpenXRFbPassthroughExtensionWrapper() {
 
 void OpenXRFbPassthroughExtensionWrapper::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("is_passthrough_supported"), &OpenXRFbPassthroughExtensionWrapper::is_passthrough_supported);
-	ClassDB::bind_method(D_METHOD("is_passthrough_enabled"), &OpenXRFbPassthroughExtensionWrapper::is_passthrough_enabled);
+	ClassDB::bind_method(D_METHOD("is_passthrough_started"), &OpenXRFbPassthroughExtensionWrapper::is_passthrough_started);
+
+	ClassDB::bind_method(D_METHOD("set_texture_opacity_factor"), &OpenXRFbPassthroughExtensionWrapper::set_texture_opacity_factor);
+	ClassDB::bind_method(D_METHOD("get_texture_opacity_factor"), &OpenXRFbPassthroughExtensionWrapper::get_texture_opacity_factor);
+
+	ClassDB::bind_method(D_METHOD("get_current_layer_purpose"), &OpenXRFbPassthroughExtensionWrapper::get_current_layer_purpose);
+
+	ClassDB::bind_method(D_METHOD("set_edge_color"), &OpenXRFbPassthroughExtensionWrapper::set_edge_color);
+	ClassDB::bind_method(D_METHOD("get_edge_color"), &OpenXRFbPassthroughExtensionWrapper::get_edge_color);
+
+	ClassDB::bind_method(D_METHOD("set_passthrough_filter"), &OpenXRFbPassthroughExtensionWrapper::set_passthrough_filter);
+	ClassDB::bind_method(D_METHOD("get_current_passthrough_filter"), &OpenXRFbPassthroughExtensionWrapper::get_current_passthrough_filter);
+	ClassDB::bind_method(D_METHOD("set_color_map"), &OpenXRFbPassthroughExtensionWrapper::set_color_map);
+	ClassDB::bind_method(D_METHOD("set_mono_map"), &OpenXRFbPassthroughExtensionWrapper::set_mono_map);
+	ClassDB::bind_method(D_METHOD("set_brightness_contrast_saturation"), &OpenXRFbPassthroughExtensionWrapper::set_brightness_contrast_saturation);
+
+	ClassDB::bind_method(D_METHOD("has_passthrough_capability"), &OpenXRFbPassthroughExtensionWrapper::has_passthrough_capability);
+	ClassDB::bind_method(D_METHOD("has_color_passthrough_capability"), &OpenXRFbPassthroughExtensionWrapper::has_color_passthrough_capability);
+	ClassDB::bind_method(D_METHOD("has_layer_depth_passthrough_capability"), &OpenXRFbPassthroughExtensionWrapper::has_layer_depth_passthrough_capability);
+
+	ADD_SIGNAL(MethodInfo("openxr_fb_projected_passthrough_layer_created"));
+	ADD_SIGNAL(MethodInfo("openxr_fb_passthrough_stopped"));
+	ADD_SIGNAL(MethodInfo("passthrough_state_changed", PropertyInfo(Variant::INT, "event_type")));
```

Comment:
Can you update this signal to `openxr_fb_passthrough_state_changed` to match the pattern of the above signals.

Reply:
Updated!